entrypoint.sh

```diff
@@ -1,22 +1,41 @@
-#!/bin/sh -l
+#!/bin/bash
```

Comment:
I am considering switching to a more _slim_ base image, but using `bash` will not be an issue

Reply:
Got it! It is probably a good idea, I made some improvements over Ebirah, I'll provide a PR which reduces the size of the Docker image.

Reply:
Here you go: https://github.com/jonasbn/ebirah/pull/46

Reply:
Thanks I will check it out